docs/installguide/uninstall.rst

```diff
@@ -11,14 +11,22 @@ _______
 Mac OSX
 _______
 
-1. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
+Option 1:
+
+1. Launch ``KA-Lite`` from your ``Applications`` folder.
 2. Click on the app icon at the menu bar.
 3. Click on ``Preferences`` in the menu option.
-4. Click the ``Reset App`` from the ``Advanced`` tab.
-5. You will be prompted that "This will reset app. Are you sure?", just click on ``OK`` button.
-6. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
-7. Quit the ``KA-Lite Monitor`` app (do not click the ``Apply`` button!).
-8. Move the ``KA-Lite Monitor`` app to ``Trash``.
+4. Click the ``Uninstall KA Lite`` from the ``Preferences`` tab.
+5. if you want to delete your KA Lite data folder checked the ``Delete KA Lite data folder`` checkbox.
+6. You will be prompted that ``Are you sure that you want to uninstall the KA Lite application?``, just click on ``OK`` button.
+7. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
+
+Option 2:
```

Comment:
Call it:

    Uninstallation from command line
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~